WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -299,6 +300,25 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
   });
 }
 
+/**
+ * @return Current user interface style as a string
+ */
++ (NSString *)userInterfaceStyle
+{
+  if (@available(iOS 13.0, *)) {
```

Comment:
Available since 12.0?
https://developer.apple.com/documentation/uikit/uiuserinterfacestyle?language=objc

Reply:
When I change to 12 then Xcode is showing me a warning about that method is available only since v13. Do you observe the same message?

Reply:
I, actually, changed the implementation to be fully dynamic because of https://travis-ci.org/appium/WebDriverAgent/jobs/647782920?utm_medium=notification&utm_source=github_status